.gitignore

```diff
@@ -2,3 +2,4 @@
 *.tar.gz
 *.zip
 *.png
+pacstall-programs/*
```

Comment:
remove this from the merge. this would break using `git add -A`

Reply:
So, I need to remove this? Or with other method?

Reply:
correct, this line shouldn't be here. just go to https://github.com/Anifyuli/pacstall-programs/edit/ship-youtube-music-deb/.gitignore?pr=%2Fpacstall%2Fpacstall-programs%2Fpull%2F5021 and remove the line